packages/nhsuk-frontend/src/nhsuk/components/file-upload/template.njk

```diff
@@ -0,0 +1,85 @@
+{% from "nhsuk/macros/attributes.njk" import nhsukAttributes %}
+{% from "nhsuk/macros/i18n.njk" import nhsukI18nAttributes %}
+{% from "nhsuk/components/error-message/macro.njk" import errorMessage %}
+{% from "nhsuk/components/hint/macro.njk" import hint %}
+{% from "nhsuk/components/label/macro.njk" import label %}
+
+{#- A record of other elements that we need to associate with the input using
```

Comment:
Minor thing. I've been copying from GOV.UK Frontend as-is (almost) in https://github.com/nhsuk/nhsuk-frontend/pull/1567 and resisting things like this. Do you want me to fix it in my PR too or change it here to match?

```suggestion
{#- a record of other elements that we need to associate with the input using
```

Reply:
Disagree. We should use capitalisation because we’re not animals. There are already changes in this template (such as the name spacing), so it’s never going to match 100%.

And if that means going through all our components to use capitalisation in comments, I’ll do it.

Reply:
Thank you! And sorry to be a grammar pedant 😊